components/04-views/dashboard/empty/empty.hbs

```diff
@@ -2,12 +2,12 @@
     {{ render '@header' }}
 
     <div class="dashboard container">
-        <div class="grid">
-            <div class="grid__column grid__column--1-4 dashboard__sidebar">
+        <div class="row">
+            <div class="col-xs-12 col-md-4 col-lg-3"></div>
```

Comment:
I'm not sure, but this closing tag is probably unnecessary

Reply:
removed